src/main.js

```diff
@@ -37,9 +40,55 @@ function fetchData(data, method, myHeaders, request) {
 var init = { method: method,
                headers: myHeaders,
                mode: 'cors',
-               body: JSON.stringify(data),
                cache: 'default' };
 
+if (method !== 'GET') init.body = JSON.stringify(data);
+
 return fetch(request, init)
         .then(response => response.json())
 }
+
+function viewEmployee(event) {
+  event.preventDefault();
+  const employeeData = new FormData(event.target);
+
+  const employee = {
+    id: employeeData.get('id')
+  };
+
+  var headers = new Headers();
+  headers.append('Content-Type', 'application/json');
+
+  const request = new Request('/viewemployee/' + employee.id);
+
+  fetchData(employee, 'GET', headers, request)
+    .then(response => {
+      if (response.error) {
+        let idUnsuccessful = response.error;
+        alert(idUnsuccessful);
+      }
+      else if (response.id) {
+        let name = document.getElementById('profile-name');
```

Comment:
Can you refactor this into a function that rebuilds the profile and swaps it into the DOM?